src/ProgressMeter.jl

```diff
@@ -316,17 +316,13 @@ function update!(p::ProgressThresh, val, color::Symbol; options...)
     update!(p, val; options...)
 end
 
-function update!(p::ProgressUnknown; options...)
-    p.counter += 1
+function update!(p::ProgressUnknown; increasecounter=true, options...)
```

Comment:
What's the `increasecounter` for?

Reply:
The idea was that you could call `update!` without increasing the counter, e.g., if you only want to update the values display in `showvalues`:
```julia
update!(progress; increasecounter=false, showvalues=[(:myKey,  myValue)])
```

But I also see now that this is somewhat duplicate to the `update!(prog, counter)` method where you would just need to keep track of the counter manually.